OracleUnifiedDirectory/samples/README.md

```diff
@@ -0,0 +1,1347 @@
+# Creating Oracle Unified Directory Docker Containers
+
+## Contents
+1. [Before You Begin](#before-you-begin)
+2. [Background](#background)
+3. [What Do You Need?](#what-do-you-need)
+4. [Directory Server/Service (instanceType=Directory)](#directory-serverservice-instancetypedirectory)
+5. [Directory Proxy Server/Service (instanceType=Proxy)](#directory-proxy-serverservice-instancetypeproxy)
+6. [Replication Server/Service (instanceType=Replication)](#replication-serverservice-instancetypereplication)
+7. [Directory Server/Service added to existing Replication Server/Service (instanceType=AddDS2RS)](#directory-serverservice-added-to-existing-replication-serverservice-instancetypeaddds2rs)
+8. [Directory Server/Service (myoudds2b) added to existing Directory Server/Service (myoudds2) (instanceType=AddDS2RS)](#directory-serverservice-myoudds2b-added-to-existing-directory-serverservice-myoudds2-instancetypeaddds2rs)
+9. [Directory Client to run CLIs like ldapsearch, dsconfig, and dsreplication](#directory-client-to-run-clis-like-ldapsearch-dsconfig-and-dsreplication)
+10. [Directory Server/Service (instanceType=Directory) with dstune configuration options](#directory-serverservice-instancetypedirectory-with-dstune-configuration-options)
+11. [Access interfaces (LDAP / LDAPS / HTTP / HTTPS) exposed by OUD container](#access-interfaces-ldap-ldaps-http-https-exposed-by-oud-container)
+12. [Removing an OUD Docker Container](#removing-an-oud-docker-container)
+
+## Before You Begin
+								
+The samples below show you how to create and configure Oracle Unified Directory (OUD) 12.2.1.4.0 Docker containers.
+
+## Background
```

Comment:
I don't think the background of Docker or containers is necessary in this repo.

Reply:
Removed.